Sources/GRPCHTTP2Core/Client/GRPCClientStreamHandler.swift

```diff
@@ -148,6 +158,30 @@ extension GRPCClientStreamHandler {
   func handlerRemoved(context: ChannelHandlerContext) {
     self.stateMachine.tearDown()
   }
+
+  func channelInactive(context: ChannelHandlerContext) {
+    switch self.stateMachine.unexpectedInboundClose(reason: .channelInactive) {
```

Comment:
Can we avoid the duplication here? e.g. `func unexpectedClose(context: ChannelHandlerContext, reason: Reason)`